src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -748,15 +772,16 @@ spec:
   phrase: ContainerSigSegvReceived
   issuer: USER_CONTAINER
   causer: USER_CONTAINER
-  type: USER_FAILURE
+  type: UNKNOWN_FAILURE
   stage: RUNNING
   behavior: PERMANENT
```

Comment:
>PERMANENT [](start = 12, length = 9)

UNKNOWN,


refer:

- code: -300
  phrase: PodFailedWithoutFailedContainer
  issuer: PAI_FC
  causer: UNKNOWN
  type: UNKNOWN_FAILURE
  stage: LAUNCHING
  behavior: UNKNOWN
  reaction: RETRY_TO_MAX
  reason: "Pod failed but without any failed container, may be stopped by K8S"
  repro:
  - "Submit a job which will be rejected by kubelet"
  solution:
  - "Check diagnostics and find root cause"
  - "Wait result from next retry"
  - "Contact Cluster Admin"

Reply:
Fixed